sdks/java/core/src/main/proto/README.md

```diff
@@ -1,7 +1,25 @@
-## Protocol Buffers in Google Cloud Dataflow
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
 
-This directory contains the Protocol Buffer messages used in Google Cloud
-Dataflow.
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+## Protocol Buffers in Apache Beam
```

Comment:
We have a few options forward here, but yes I think that this can wait for another PR.

Right now we have not yet re-deployed the proto jar under `org.apache.beam`